src/components/Popups/index.tsx

```diff
@@ -65,16 +69,24 @@ export default function Popups() {
   const { chainId } = useActiveWeb3React()
   const isNotOnMainnet = Boolean(chainId && chainId !== SupportedChainId.MAINNET)
 
+  // donation popup, hide if after fixed duration
+  const [donationVisible] = useShowDonationLink()
+  const timestamp = useCurrentBlockTimestamp()
+  const durationOver = timestamp ? timestamp.toNumber() > DONATION_END_TIMESTAMP : false
+  const showDonation = donationVisible !== false && !durationOver
+
   return (
     <>
       <FixedPopupColumn gap="20px" extraPadding={urlWarningActive} xlPadding={isNotOnMainnet}>
         <ClaimPopup />
         {activePopups.map((item) => (
           <PopupItem key={item.key} content={item.content} popKey={item.key} removeAfterMs={item.removeAfterMs} />
         ))}
+        {showDonation ? <DonationLink /> : null}
```

Comment:
Why does DonationLink not get pushed to activePopups?

Reply:
active popups are from transactions , this one is a custom element